sample/adder/gradlew

```diff
@@ -0,0 +1,249 @@
+#!/bin/sh
```

Comment:
Do you really need all this gradle stuff in a sample directory? It looks a bit scary. I thought user would provide just Perform.java and maybe one build.gradle.kts file

Reply:
Look what I've discovered. I deleted a few gradle files and it still worked :smile:  I left these:
```
.
├── app
│   ├── build.gradle.kts
│   └── src
│       └── main
│           └── java
│               └── Perform.java
├── job.yaml
├── Makefile
└── settings.gradle.kts
```

This is because job-template.Dockerfile has:
```
FROM gradle:jdk21 as build
...
RUN gradle war
```
You use `gradle` binary from a docker image (not `./gradlew`), so you don't need gradlew at all. As far as I remember, gradlew is what virtualenv is to Python. gradlew is a script that downloads a local copy of gradle so that you don't need to install it. Not needed if you already have it system-wide.

Reply:
Great discovery. They wrote in docs that ./gradlew and wrapper should be in repo, so that whover downloads it on whatever system, will be able to build it locally without downloading everything. Guess here we have a different setup, so we can do it differently.

OTOH, having those files in repo, allows to build the app directly, without a need for RT. I think let's wait for feedback from users, and for now let's go with simplified setup.